src/Tooltip/Tooltip.tsx

```diff
@@ -113,7 +114,13 @@ export default class Tooltip extends React.Component<Props> {
         rootClose
         trigger={hide ? [] : ["focus", clickTrigger ? "click" : "hover"]}
       >
-        {children}
+        <span
+          className={cssClass.FOCUSABLE_TRIGGER}
+          onMouseDown={e => e.preventDefault()}
+          tabIndex={0}
```

Comment:
Can we also add `aria-describedby={this.id}` here?

Reply:
added! thanks for catching that!